lib/src/components/NxThreatIndicator/__tests__/NxThreatIndicator.test.tsx

```diff
@@ -4,116 +4,115 @@
+    expect(threatIndicator).toBeInTheDocument();
   });
 
-  it('sets the modifier class using the threatLevelCategory if it is provided', function() {
-    const icon = getShallowComponent({ threatLevelCategory: 'low' }).find(NxFontAwesomeIcon);
-
-    expect(icon).toMatchSelector('.nx-threat-indicator--low');
-    expect(icon).not.toMatchSelector('.nx-threat-indicator--unspecified');
-  });
+  it('takes precedence of threatLevelCategory if both props are provided', async function() {
+    const el = quickRender({ policyThreatLevel: 9, threatLevelCategory: 'low' }),
+        user = userEvent.setup(),
+        threatIndicator = el.getByRole('img', { hidden: true, queryFallbacks: true })!;
 
-  it('sets the modifier class by converting the policyThreatLevel if it is provided', function() {
-    expect(getShallowComponent({ policyThreatLevel: 0 }).find('.nx-threat-indicator'))
-        .toMatchSelector('.nx-threat-indicator--none');
-    expect(getShallowComponent({ policyThreatLevel: 1 }).find('.nx-threat-indicator'))
-        .toMatchSelector('.nx-threat-indicator--low');
-    expect(getShallowComponent({ policyThreatLevel: 3 }).find('.nx-threat-indicator'))
-        .toMatchSelector('.nx-threat-indicator--moderate');
-    expect(getShallowComponent({ policyThreatLevel: 5 }).find('.nx-threat-indicator'))
-        .toMatchSelector('.nx-threat-indicator--severe');
-    expect(getShallowComponent({ policyThreatLevel: 9 }).find('.nx-threat-indicator'))
-        .toMatchSelector('.nx-threat-indicator--critical');
-    expect(getShallowComponent({ policyThreatLevel: 9 }).find('.nx-threat-indicator'))
-        .not.toMatchSelector('.nx-threat-indicator--unspecified');
-  });
-
-  it('sets the modifier class using the threatLevelCategory if both props are provided', function() {
-    const component = getShallowComponent({ policyThreatLevel: 9, threatLevelCategory: 'low' });
-    const icon = component.find(NxFontAwesomeIcon);
-
-    expect(icon).toMatchSelector('.nx-threat-indicator--low');
-    expect(icon).not.toMatchSelector('.nx-threat-indicator--critical');
-    expect(icon).not.toMatchSelector('.nx-threat-indicator--unspecified');
+    await user.hover(threatIndicator);
+    const tooltip = await screen.findByRole('tooltip');
+    expect(tooltip).toHaveTextContent('Low');
   });
 
   it('adds aria attrs to help the icon show up for screen readers', function() {
-    const component = getShallowComponent({ policyThreatLevel: 9, threatLevelCategory: 'low' });
+    const el = quickRender({ policyThreatLevel: 9, threatLevelCategory: 'low' }),
+        threatIndicator = el.getByRole('img', { hidden: true, queryFallbacks: true })!;
 
-    expect(component.find(NxFontAwesomeIcon)).toHaveProp('aria-label', 'threat level low');
-    expect(component.find(NxFontAwesomeIcon)).not.toHaveProp('aria-hidden');
+    expect(threatIndicator).toHaveAttribute('aria-label', 'threat level low');
```

Comment:
`toHaveAccessibleName` would be better

Reply:
> `toHaveAccessibleName` would be better

Changed here: ce9d721aaf8ea9aed78dfad31a331f3577308de1